plugin.video.montreal.greek-tv/resources/language/English/strings.po

```diff
@@ -0,0 +1,39 @@
+# Addon Name: Montreal Greek TV
```

Comment:
please adopt the new layout for language folders:

`plugin.video.montreal.greek-tv/resources/language/resource.language.en_gb/strings.po`

`plugin.video.montreal.greek-tv/resources/language/resources.language.el_gr/strings.po`